README.md

```diff
@@ -16,59 +18,67 @@ transformers:
   - sass
 ```
 
-After adding the transformer your `.sass` and `.scss` files will be automatically transformed to
+After adding the transformer all your `.sass` and `.scss` files will be automatically transformed to
 corresponding `.css` files.
 
-You need to have [Sass](http://sass-lang.com/) installed and available on the path.
-
-## Configuration
-
-You can also pass options to Sass if necessary:
+If your main file imports other files you will need to add the option `include_paths`,
+ to let `sass` know which folder will be used for processing imports:
 
 ```yaml
+dependencies:
+  sass: any
 transformers:
   - sass:
-      executable: /path/to/sass     # Sass executable to use
-      compass: true                 # Include compass
-      line-numbers: true            # Include line numbers in output
-      style: compact                # Style of generated CSS
-      copy-sources: true            # Copy original .scss/.sass files to output directory
+      entry_points: path/to/main.scss
+      include_paths: path/to/folder/with/other/scss
 ```
 
-## Using SassC
-
-You can use [SassC](https://github.com/hcatlin/sassc) instead of normal Sass by specifying executable
-as 'sassc' (or any path ending with 'sassc'):
+you can have multiple `entry_points` and multiple `include_paths`:
 
 ```yaml
+dependencies:
+  sass: any
 transformers:
   - sass:
-      executable: sassc  # or /path/to/sassc
+      entry_points:
+        - path/to/main.scss
+        - path/to/other.scss
+        - included/*.scss       # this include all scss files inside included
+        - !exlcuded/*.scss      # this exclude all scss files insde excluded
+      include_paths:
+        - path/to/folder/with/other/scss1
+        - path/to/folder/with/other/scss2
 ```
 
-SassC only supports `.scss`-files and does not support Compass.
+> Doing `pub serve` during development, css files are going to live in memory only. When you do `pub build` actual css files will be output in build folder
```

Comment:
I would change this to "By using `pub serve` ...". "Executing `pub build` creates actual css files in the build folder.